packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/en.ftl

```diff
@@ -0,0 +1,17 @@
+## ConfirmSignupCode page
+## Users see this page after they have initiated account sign up,
+# and a confirmation code has been sent to their email address.
+
+# String within the <span> element appears on a separate line
+# If more appropriate in a locale, the string within the <span>, "for your { -product-firefox-account }"
+# can stand alone as "{ -product-firefox-account }"
+confirm-signup-code-heading = Enter confirmation code<span> for your { -product-firefox-account }</span>
```

Comment:
nit: not sure if the `<span>` should be aligned with the `code <span>for` versus `code<span> for`:

```suggestion
confirm-signup-code-heading = Enter confirmation code <span>for your { -product-firefox-account }</span>
```

Reply:
Checking against most of the other PRs with the `<span>` subheading, the space does in fact seem to come before.